app/build.gradle

```diff
@@ -120,7 +120,7 @@ dependencies {
     provided "frankiesardo:icepick-processor:$icepickVersion"
 
     compile "com.jakewharton:butterknife:$butterKnifeVersion"
-    annotationProcessor "com.jakewharton:butterknife-compiler:$butterKnifeVersion"
+    kapt "com.jakewharton:butterknife-compiler:$butterKnifeVersion"
```

Comment:
kapt is kotlin annotation processor. It knows how to read annotations in kotlin and java classes. APT on the other hand doesn't.